embulk-standards/src/main/java/org/embulk/standards/LocalFileInputPlugin.java

```diff
@@ -148,6 +165,16 @@ public FileVisitResult preVisitDirectory(Path path, BasicFileAttributes attrs)
                 @Override
                 public FileVisitResult visitFile(Path path, BasicFileAttributes attrs)
                 {
+                    try {
```

Comment:
The actual goal of this part looks avoiding directories from listing, even if they are symbolic links or not. Not checking whether they are symbolic links. Right?

How about making this comment like:

```
// Avoid directories from listing.
// Directories are normally unvisited with |FileVisitor#visitFile|, but symbolic links to
// directories are visited like files unless |FOLLOW_LINKS| is set in |Files#walkFileTree|.
// Symbolic links to directories are explicitly skipped here by checking with |Path#toReadlPath|.
```